ssh2/utils.c

```diff
@@ -7,8 +7,8 @@
 #elif PY_VERSION_HEX < 0x02060000 || (0x03000000 <= PY_VERSION_HEX && PY_VERSION_HEX < 0x03030000)
     #error Cython requires Python 2.6+ or Python 3.3+.
 #else
-#define CYTHON_ABI "0_29_6"
-#define CYTHON_HEX_VERSION 0x001D06F0
+#define CYTHON_ABI "0_29_7"
```

Comment:
Is there some possibility for problems due to this version being different?

Reply:
Looking at the changes made to these files I don't see many problem possibility, but to be sure I should install the older cython and recompile.

Reply:
Done